ui/webui/test/machine_install.py

```diff
@@ -87,7 +96,7 @@ def start(self):
                 "-device virtio-net-pci,netdev=hostnet0,id=net0,addr=0x4' "
                 f"--initrd-inject {os.getcwd()}/test/ks.cfg "
                 "--extra-args 'inst.ks=file:/ks.cfg' "
-                "--disk size=15,format=qcow2 "
+                f"--disk path={disk_image},bus=virtio,cache=unsafe,format=qcow2 "
```

Comment:
Did you consider just keeping the code as is and instead catching the exception and retrying it 2-3 times in case it collides with another VM image creation?
Just an alternative suggestion to consider, not review request.

Reply:
Yes, I considered it, but given Cockpit is pre-creating the image as well in virtual_machine.py, and we are pre-creating the image in Anaconda kickstart tests I was thinking this is a better way to go.